package.json

```diff
@@ -30,6 +30,7 @@
     "@adobe/parcel-plugin-htl": "^0.9.4",
     "@adobe/petridish": "^1.6.0",
     "archiver": "^3.0.0",
+    "bluebird": "^3.5.2",
```

Comment:
noooo :-)

Reply:
Bluebird is already in the dependency chain, I've just moved it to top-level.

Reply:
> Bluebird is already in the dependency chain, I've just moved it to top-level.

```
$ npm ls bluebird
@adobe/helix-cli@0.8.1-pre.14 /Users/stefan/projects/helix-cli
├─┬ @adobe/hypermedia-pipeline@0.6.0
│ ├── bluebird@3.5.2
│ └─┬ tmp-promise@1.0.5
│   └── bluebird@3.5.2  deduped
└─┬ request-promise@4.2.2
  └── bluebird@3.5.2  deduped
```

`request-promise-native` can be simply replaced with `request-promise-native`.

`@adobe/hypermedia-pipeline` would need to be ported, replacing `tmp-promise` with a module that doesn't include a bluebird dependency and removing the direct `bluebird` dependency. 

i am +1 for using native Promise unless there's a really good reason for using a Promise library..

Reply:
me, too :-) especially since the bluebird magic in hypermedia pipeline is hard to understand :-)